packages/analytics/dashboard-renderer/src/components/QueryDataProvider.vue

```diff
@@ -96,7 +96,7 @@ const { data: v4Data, error, isValidating } = useSWRV(queryKey, async () => {
     emit('queryComplete')
   }
 }, {
-  refreshInterval: 0,
+  refreshInterval: props.context.refreshInterval ?? 0,
```

Comment:
Let's set the default to 30 seconds, based on the argument that we usually want it in order to showcase real-time data.